advice_for_future_giraf/PO_advice.md

```diff
@@ -0,0 +1,151 @@
+Another reason to document as much as possible, is that the users' feedback is the basis of all the user stories you are going to create.
+
+## Creating user stories
+As the PO group it is your responsibility to create user stories.
+User stories are created based on requests from the users.
+We structured user stories in the following way:
+- As a ... I would like ..., so that ...
+
+When creating a user story you should consider the amount of work that is needed for it to be completed, and whether or not it should be split up into multiple smaller user stories.
+We created user stories in the weekplanner repository on our GitHub page under the issue tab.
+This made it easy to organize as user stories are uniquely numbered and can be put into milestones.
+When creating a pull request it is easy to tag the user story so that it will be automatically closed when the pull request has been merged.
+It also allowed us to assign them with tags such as "feature", and give them different priorities ranging from lowest to highest.
+A user story should, if needed, contain a prototype that ideally has been approved by a customer and a further description of the problem.
+We also had great success having one of the more experienced Flutter developers write a short technical comment explaining how they would structure the solution for the user story.
+This gave the developers who were not as experienced with Flutter a better starting point.
+
+Below is an image of a user story.
+
+![User Story](images/userStory.PNG)
+
+As you can see on the figure, the user story has the number 88 and was created by the user Eduardsen.
+It has the tags "priority: highest" and "type: feature" and is under the milestone called sprint 2.
+In the right hand side you can see the profile picture of the developer currently assigned to implementing this user story.
+Another approach to creating user stories is when developer groups create feature requests.
+In most cases these need to be rewritten into user stories if the problem they are describing has not been formulated correctly or adequately.
+When creating user stories, take special care to ensure that your user stories are not written ambiguously and, if so, that you have the precise functionality that the users wanted explained within them.
+
+## Prototypes
+One of the important tasks of the PO group is to create and maintain prototypes.
+Prototypes should conform to the design guide which can be found on the Github Wiki-page.
+PO groups from previous years had been using PowerPoint to create prototypes, but we found this process to be slow and require a lot of repetitive work.
+Instead, we opted to use Adobe XD which reduced repetitive work and let us import the exact icons we used throughout the design guide, making the prototypes much more consistent.
```

Comment:
What if they don't know that? Do you have anything that can get them started, or should they pick something they are familiar with?

Reply:
What do you mean? We're sharing the adobe XD file with the prototypes - but I just noticed that there's no link to it, which will be fixed asap.

Reply:
Fixed